sqlmesh/core/constants.py

```diff
@@ -70,6 +70,7 @@
 
 EXTERNAL_MODELS_YAML = "external_models.yaml"
 EXTERNAL_MODELS_DEPRECATED_YAML = "schema.yaml"
+REQUIREMENTS = "sqlmesh.lock"
```

Comment:
Not sure I like this name. In context of SQLMesh it's pretty ambiguous. Why not `requirements.sqlmesh.txt`? Or something like that.

Reply:
i'll make it sqlmesh-requirements.lock because it's not a requirements file, it's a lock file. the syntax is different